common/constants/custom_panels.ts

```diff
@@ -9,7 +9,21 @@
  * GitHub history for details.
  */
 
-export const CUSTOM_PANELS_API_PREFIX = '/api/observability/custom_panels';
+export const CUSTOM_PANELS_API_PREFIX = '/api/observability/operational_panels';
 export const CUSTOM_PANELS_DOCUMENTATION_URL = 'https://www.opensearch.org';
 export const CREATE_PANEL_MESSAGE = 'Enter a name to describe the purpose of this custom panel.';
-export const DATE_FORMAT = 'MM/DD/YYYY hh:mm A';
+export const DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss';
```

Comment:
i don't think UI displays date with year month date, is this used by PPL? if so could you add a comment or add "PPL" in the variable name?

Reply:
Yes this is the PPL format will edit the name and move it to shared constants.